packages/block-library/src/index.native.js

```diff
@@ -143,6 +144,8 @@ export const registerCoreBlocks = () => {
 		quote,
 		// eslint-disable-next-line no-undef
 		typeof __DEV__ !== 'undefined' && __DEV__ ? mediaText : null,
+		// eslint-disable-next-line no-undef
+		typeof __DEV__ !== 'undefined' && __DEV__ ? group : null,
```

Comment:
I know there was already one before this, but I'm not sure I understand what the `typeof __DEV__ !== 'undefined'` is trying to achieve. Wouldn't `__DEV__` or `!! __DEV__` do the same thing?

Reply:
I was wondering the same thing and agree that `__DEV__` or `!!__DEV__` will do the same, but decided to keep the convention, however, this condition can be simplified.